hack/deploy.sh

```diff
@@ -130,12 +133,19 @@ then
     "${CMD}" get crd ${HCO_CRD_NAME} -o go-template='{{ range .status.conditions }}{{ .type }}{{ "\t" }}{{ .status }}{{ "\t" }}{{ .message }}{{ "\n" }}{{ end }}'
 fi
 
+# note that generated certificates are necessary for webhook deployments
+# manifest-templator does not add them into operator.yaml at the moment. 
+# when a new webhook (deployed by OLM in production) is introduced, 
+# it must be added into webhooks.yaml as well.
+echo "Creating resources for webhooks"
+"${CMD}" apply -f _out/webhooks.yaml
+
 if [ "${CI}" != "true" ]; then
-	"${CMD}" create -f _out/operator.yaml
+	"${CMD}" apply -f _out/operator.yaml
 else
 	sed -E 's|^(\s*)- name: KVM_EMULATION$|\1- name: KVM_EMULATION\n\1  value: "true"|' < _out/operator.yaml > _out/operator-ci.yaml
 	cat _out/operator-ci.yaml
-	"${CMD}" create -f _out/operator-ci.yaml
+	"${CMD}" apply -f _out/operator-ci.yaml
 fi
 
 # Wait for the HCO to be ready
```

Comment:
Maybe we should also start explicitly checking the status of other webhooks here

Reply:
Hi @tiraboschi . Thanks for checking the PR. Could you please elaborate these two words in your comment?

1) "status" -> did you mean the status of deployments or status of validatingwebhookconfiguration CRs ?
2) "other" -> other than validating? other than hco? other than hco + node-maintenance?

Reply:
yes, sorry.
As on line 144/148 but for hco webhooks and node-maintenance deployments

Reply:
Check for node-maintenance-operator is available in the for loop here https://github.com/kubevirt/hyperconverged-cluster-operator/pull/1001/commits/74a9f3a6e33f643a2926bdf09977d256bb92c7ba#diff-012508938ec4e97ecf905766cea58011d7b8a1da6b7f82d362ffa10220b33184L158

I just added a wait for hco-webhook as well.